src/Prefabs/header.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+    echo '<header>
```

Comment:
this doesnt have to be echo'd - just remove the php tags and have the html code directly in the file